UtopiaCity/Controllers/CityAdministration/RequestsConfirmationController.cs

```diff
@@ -0,0 +1,88 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using UtopiaCity.Models.Sport;
+using UtopiaCity.Services.Sport;
+using UtopiaCity.ViewModels.Sport;
+
+namespace UtopiaCity.Controllers.CityAdministration
+{
+    public class RequestsConfirmationController : Controller
+    {
+        private readonly RequestToAdministrationService _requestToAdministrationService;
+        private readonly SportComplexService _sportComplexService;
+        private readonly IMapper _mapper;
+
+        public RequestsConfirmationController(RequestToAdministrationService requestToAdministrationService, SportComplexService sportComplexService, IMapper mapper)
+        {
```

Comment:
Changes from other branch?

Reply:
Yes, to test requests confirmation from sports complexes I merged sports branch into mine.